website/docs/tutorial/1-setting-up.md

```diff
@@ -73,45 +73,6 @@ final as (
 select * from final
 ```
 
-## Create a BigQuery project
```

Comment:
suggestion: for the video on Line 10, I think we can keep it, but can we chop off the last 3 seconds of the video that says “First off, we will be creating a BigQuery Project”?

Reply:
sounds good to me 👍

Reply:
How do we do that?

Reply:
a couple options:
Option 1: Jump into loom and trim the final 3 seconds.  I don't know if Claire transferred this to a public dbt Labs space though

Option 2: download the video, edit in Descript, upload to Wistia, and replace the video.

Reply:
Hmm i don't see the option - do you? https://www.loom.com/share/cb99861ab1034f7fab5fa48529e61f85 I also just realized I was using either fishtown or personal google to record all my videos previously? Need to look into that.

But I tried route 2, and created this. Not sure what to do with it now though -- what html/css do i use to input Wistia?
https://dbtlabs-1.wistia.com/medias/jh4o7fawrd

Reply:
check out the very subtle raw code here from Krevitt: https://raw.githubusercontent.com/dbt-labs/docs.getdbt.com/current/website/docs/tutorial/refactoring-legacy-sql.md

There is a very subtle HTML tag where you replace the ID from the video

`<WistiaVideo id="5u67ik9t66" />`

I imagine that is the code at the end of your URL